app/models/reservation_scopes.rb

```diff
@@ -50,6 +50,9 @@ def self.included(base) # rubocop:disable MethodLength, AbcSize
       scope :checkoutable, lambda {
         where('start_date <= ?', Time.zone.today).reserved
       }
+      scope :future, lambda {
```

Comment:
Which should? The `pending` scope should include all `untouched` reservations between today and 8 days from now. That will, by default, include all checkoutable reservations for today.